.gitignore

```diff
@@ -1,3 +1,6 @@
+netlify.toml.bak
```

Comment:
Do we need to add `.env` to the gitignore? I don't see it here

Reply:
It's in there, buried deep on line 75 :D